features/http2.md

```diff
@@ -0,0 +1,15 @@
+---
+title: HTTP/2
+category: network
+bugzilla: 1047594
+firefox_status: 34
+spec_url: https://http2.github.io/
+spec_repo: https://github.com/http2/http2-spec
+chrome_ref: 5152586365665280
+webkit_status: shipped
+webkit_footnote: Depends on the networking stack of the system (https://lists.webkit.org/pipermail/webkit-help/2015-February/003884.html).
+ie_ref: HTTP/2
+caniuse_ref: http2
+---
+
+The second version of the HTTP protocol.
```

Comment:
The description could need some pizzazz, maybe like: "Next revision of the HTTP protocol for faster transport of web content".